### PR TITLE
docs: Explain how to create certificate chain file

### LINKED ISSUE #4010: Support Certificate chain while configuring SSL
## Expected Behavior
A certificate chain file named public.crt should be recognised and HTTPS on Minio server should work. 

## Current Behavior
Minio server returns an error about a mismatch between private and public keys.
`time="2017-03-30T17:36:41Z" level=fatal msg="Failed to start minio server." cause="tls: private key does not match public key" source="[server-main.go:580:serverMain.func1()]"
`
## Steps to Reproduce (for bugs)

1. Add private.key and public.crt file 
2. public.crt file should contain multiple `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` stanzas. I tested with 4.
3. restart minio docker container so that the new certificates are recognised
4. `docker logs -f minio`

## Your Environment

* Version used: 2017-03-16T21:50:32Z
* Environment name and version (e.g. nginx 1.9.1): Docker version 1.12.6 
* Server type and version: VMware virtual machine
* Operating System and version: CentOS Linux release 7.3.1611 (Core)
* Link to your project:

### PR DESCRIPTION
**Pull request by vadmeste**
## Description
public.crt needs sometimes to have a chain certificate, this PR
explains how to construct public.crt when certificate are issued
by a certificate authority.

## Motivation and Context
Fixes https://github.com/minio/minio/issues/4010

## How Has This Been Tested?
NA

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by justinclift** [comment]
As a thought in that doc, aren't 1024 bit keys considered "too short" for good security practise these days?

File: docs/tls/README.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ In this document, we will configure Minio servers with TLS certificates for both
 
 * Download Minio server from [here](https://docs.minio.io/docs/minio-quickstart-guide)
 
-## 2. Generate TLS certificate
+## 2. Configure with existing certificates
+
+Assuming that you are already having private and public certificates, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively.
+
+If the certificate is signed by a certificate authority, `public.crt` should be the concatenation of the server's certificate, any intermediates, and the CA's certificate.
```
Comment (justinclift) [inline docs/tls/README.md]: Maybe "CA's certificate" should be "CA's root certificate", for that extra bit of clarity?

Comment (nitisht) [reply docs/tls/README.md]: I agree..

**Comment by nitisht** [comment]
Other than one comment, LGTM.

**Comment by vadmeste** [comment]
Comments are addressed.

**Comment by justinclift** [comment]
LGTM. :smile:

**Review by nitisht — APPROVED** [review]

**Pull request merged**